### PR TITLE
Support u8 to fp16 input convert

### PR DESCRIPTION
**Pull request by ZongHong-Lyu**
Added conv command v1 to support u8 input format

### CODE REVIEW

File: include/dmp_dv_cmdraw_v1.h [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#ifndef _DMP_DV_CMDRAW_V1_H_
+#define _DMP_DV_CMDRAW_V1_H_
+
+#include "dmp_dv.h"
+
+/// @brief Raw command for convolutional block version 0.
+struct dmp_dv_cmdraw_conv_v1 {
+  struct dmp_dv_cmdraw header;  // General structure information
+
+	struct dmp_dv_buf u8tofp16_table;
```
Comment (YuyaKobayashi) [inline include/dmp_dv_cmdraw_v1.h]: Sorry, fix indentation

Comment (ZongHong-Lyu) [reply include/dmp_dv_cmdraw_v1.h]: Fixed.

File: include/dmp_dv_cmdraw_v1.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+/// @brief Raw command for convolutional block version 0.
+struct dmp_dv_cmdraw_conv_v1 {
+  struct dmp_dv_cmdraw header;  // General structure information
+
+  struct dmp_dv_buf u8tofp16_table;
+  uint16_t to_bgr;
+  uint16_t rsvd[3];
+
+  struct dmp_dv_cmdraw_conv_v0 conv_cmd;
```
Comment (ajkxyz) [inline include/dmp_dv_cmdraw_v1.h]: As `dmp_dv_cmdraw_conv_v0` is declared in `dmp_dv_cmdraw_v0.h` it is better to include it instead of `dmp_dv.h`.

Comment (ZongHong-Lyu) [reply include/dmp_dv_cmdraw_v1.h]: Fixed.

**Pull request merged**